src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -2,11 +2,19 @@
 
 import static java.lang.Math.max;
 import static java.util.stream.Collectors.joining;
+
 import static org.folio.circulation.support.JsonPropertyWriter.write;
 import static org.folio.circulation.support.JsonStringArrayHelper.toStream;
 
 import java.util.Optional;
 
+import com.neovisionaries.i18n.CountryCode;
+
+import io.vertx.core.json.JsonObject;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
```

Comment:
Why have these imports moved?

If this was done automatically by an IDE (IntelliJ 2019 recently reset these settings on an upgrade) please configure it to use the provisional standard in https://issues.folio.org/browse/FOLIO-1667

Reply:
Yes, I'm using foilo.importorder file from that link, in that file org.folio.* is at the end so Intellij moves everything else above org.folio . The format of this file is not very clear to me, but it seems to work